TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java

```diff
@@ -26,7 +27,6 @@ public void createNewProfile(String name, String type, String relatedData) {
         txtName.enter(name);
         cbbItemType.select(type);
         cbbRelatedData.select(relatedData);
-        btnFinish.click();
```

Comment:
why do you remove this?

Reply:
because when click finish, page will navigate to data profiles, not display fields

Reply:
will this affect other tests?

Reply:
affected tests have been fixed. Please check again